glommio/src/sys/uring.rs

```diff
@@ -1210,7 +1210,7 @@ macro_rules! flush_rings {
     ($( $ring:expr ),+ ) => {{
         let mut ret = 0;
         $(
-            ret += $ring.consume_submission_queue()?;
+            ret += $ring.consume_submission_queue().or_else(Reactor::busy_ok)?;
```

Comment:
isn't EAGAIN fine too ?

Reply:
That and `EINTR` I have yet to see. This version panics if it sees an unexpected error code, and I haven't seen any so far. That doesn't mean they don't happen, of course... 

I'll probably add them eventually. I just want to make our error handling is sound.

Reply:
ok, but "busy_ok" only tests for EBUSY. Your code will end up in an `except` in those cases now, no ?

Reply:
Yeah, I need to change `busy_ok` to include the other, or add new fn and chain them to handle all cases.

Reply:
Fixed in 167a03139c744a68b27dbc6dafc4a9470c9248c5